package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "textcomplete",
-  "version": "0.8.0",
+  "version": "0.8.1",
```

Comment:
Done. I only had it so that test would run on Travis, but that's no longer necessary :+1: